composer.json

```diff
@@ -22,8 +22,8 @@
     ],
     "require": {
         "php": ">=5.4.8",
-        "symfony/event-dispatcher": "~2.3",
-        "symfony/options-resolver": "~2.3",
+        "symfony/event-dispatcher": "^2.3||^3.0",
```

Comment:
Also switching to caret here, as it’s the [recommended operator](https://getcomposer.org/doc/articles/versions.md#caret).